src/main/java/com/pusher/client/Pusher.java

```diff
@@ -284,6 +287,32 @@ public PrivateChannel subscribePrivate(final String channelName, final PrivateCh
         return channel;
     }
 
+
+    /**
+     * Subscribes to a {@link com.pusher.client.channel.PrivateEncryptedChannel} which
+     * requires authentication.
+     *
+     * @param channelName The name of the channel to subscribe to.
+     * @param listener A listener to be informed of both Pusher channel protocol events and subscription data events.
+     * @param eventNames An optional list of names of events to be bound to on the channel. The equivalent of calling {@link com.pusher.client.channel.Channel#bind(String, SubscriptionEventListener)} one or more times.
```

Comment:
The new code seems to have decent horizontal size aiding readability which is great. It would be also great if the docs wouldn't exceed 100 chars (industry-wide standard). Currently it's very hard to read it in GH as it's not fitting the viewport. Even in the IDE on a massive screen it's hard to read such long lines and comfortably jump to the next line without increased risk of missing it.

Reply:
Yeah, I had mixed feelings about this. I decided it was best to keep consistent with the rest of the doc for now - and add a note to review max line lengths and refactor everything at once.

Reply:
Refactoring everything at once is hard. I think that as soon as something is identified as a tech debt we should first of all aim to not escalate it going forward.

Reply:
Hmm. That's a valid point. Let's not make the world worse. I'll go through my current diff and ensure nothing looks more than 100 characters.

Reply:
Have updated everything I think that can be wrapped 👍